lib_device_control/host/device_access_usb.c

```diff
@@ -35,6 +35,7 @@ static const int sync_timeout_ms = 500;
 void debug_libusb_error(int err_code)
 {
 #if defined _WIN32
```

Comment:
can we improve this?

Reply:
if we implement the changes suggested in https://xmosjira.atlassian.net/browse/LSM-131,  the file should get simpler.